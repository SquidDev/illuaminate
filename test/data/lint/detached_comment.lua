-- config: (lint (only doc:detached-comment))

--- This is a detached documentation comment


--- This is attached to something
local x = 0

local x = {
  x = 0, --[[-useful]] --- detached
}

--- Also detached
