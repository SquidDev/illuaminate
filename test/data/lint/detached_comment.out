detached_comment.lua:[3:1-3:44]: Detached doc comment, this will not be processed [doc:detached-comment]
   │
 3 │ --- This is a detached documentation comment
   │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
detached_comment.lua:[10:24-10:35]: Detached doc comment, this will not be processed [doc:detached-comment]
    │
 10 │   x = 0, --[[-useful]] --- detached
    │                        ^^^^^^^^^^^^
detached_comment.lua:[13:1-13:17]: Detached doc comment, this will not be processed [doc:detached-comment]
    │
 13 │ --- Also detached
    │ ^^^^^^^^^^^^^^^^^
No errors and 3 warnings
